UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/AtmosSimulation.cs

```diff
@@ -64,7 +64,10 @@ public void Run()
 		private void Update(MetaDataNode node)
 		{
 			nodes.Clear();
-			nodes.Add(node);
+
+			//Gases are frozen within walls and isolated tiles (closed airlocks) so dont do gas equalising
+			var doEqualise = node.IsOccupied == false && node.IsIsolatedNode == false;
```

Comment:
isn't this stuff handled by neighbour calculations? E.G tile is neighbour to a, but not to b  on the other side?

Reply:
> isn't this stuff handled by neighbour calculations? E.G tile is neighbour to a, but not to b on the other side?

I had to refactor that a while ago for the heat conductivity stuff, so nodes will always have all of their neighbours.